src/pages/index.jsx

```diff
@@ -7,6 +7,7 @@ import { Flex, Grid, Heading, Stack, Box, Link, Text } from "@chakra-ui/layout"
 import { Helmet } from "react-helmet"
 import SEO from "../components/Seo"
 import { RiGithubFill } from "react-icons/ri"
+import { FaRss } from "react-icons/fa"
```

Comment:
Don't mean to be super nitpicky but the rest of the site uses remix-icons `react-icons/ri` as the icon pack so it might look more consistent to go with that instead

```suggestion
import { RiRssFill } from "react-icons/ri"
```